objectRecognitionSource/README.md

```diff
@@ -279,6 +279,95 @@ Options available for all Queries (not prepended by anything) are:
     instead of only the objects recognized. Note that the data will be the sole occupant of a 1-element array when
     received, instead of being immediately available as a JSON object. This is because Query results are mandated
     to be arrays. Default is false.
+    
+Unique query capabilities available for the YOLO Processor are:
+
+*   **NOTE:** All YOLO Processor queries must have an "operation" parameter specified. If this parameter is not specified, it 
+will be set to the default value, "processNextFrame".
+
+*   **Upload images to VANTIQ:**
+    *   The user can specify an image, or multiple images to be uploaded to VANTIQ as a document. The images will be those 
+    that are saved in the output directory, which is defined in the source configuration.
+    *   Parameters:
+        *   "operation": Required. Must be set to "upload".
+        *   *Only one of the following two values *MUST* be specified, or a query error will be returned. If both values are 
+        specified, the imageName value will be used. (For optimal use, only set one of these values.)*
+            *   "imageName": A string value representing the name of the file to be uploaded. If set to "all", then all images 
+            in the output directory will be uploaded.
+            *   "imageDate": A list containing two strings, a start date and an end date. All locally saved images falling 
+            between the start and end date, (inclusive), will be uploaded. Dates must be formatted in the following 
```

Comment:
need to define the "-" date.

Also, would it be better to do "all" as` imageDate: [-,-]`? rather than a special name (that's what we really do anyway :-)

Reply:
Note that according to the tests, that's how it's done.  We don't seem to have a test for "all" as the imageName.  So maybe we should just remove it.  Use the range of _beginning of time_ to _end of time_ instead.

If we _do not_ make this change, we need to add test for `imageName: "all"`

Reply:
I'm addressing all of your other comments now. We actually do have a test that checks imageName: "all".  It is called `testImageNameUploadAll()`. In any case, I will definitely add the "-" date.

I can remove the "all" behavior entirely if that will simplify things, let me know if you think that's worth doing.

Reply:
yes, I missed that test.  Sorry.

But I think it might be simpler & clearer to just use the date range of negative-positive infinity instead of "all"